src/python/pants_release/start_release.py

```diff
@@ -200,21 +257,84 @@ def splice_into_file(release_info: ReleaseInfo, formatted: Formatted) -> None:
         # default content if the file doesn't exist yet
         existing_contents = f"# {release_info.slug} Release Series\n"
 
-    file_name.write_text(splice(existing_contents, formatted.external))
+    file_name.write_text(splice(str(release_info.version), existing_contents, formatted.external))
 
 
-def main() -> None:
-    args = create_parser().parse_args()
-    release_info = ReleaseInfo.determine(args.new)
+def update_changelog(release_info: ReleaseInfo) -> Formatted:
     branch_sha = git_fetch(release_info.branch)
     date = datetime.date.today()
-    entries = [prepare_sha(sha) for sha in relevant_shas(args.prior, branch_sha)]
+    entries = [prepare_sha(sha) for sha in relevant_shas(branch_sha)]
 
     formatted = format_notes(release_info, entries, date)
     splice_into_file(release_info, formatted)
-    print(f"\nCommit {release_info.notes_file_name()} and create a PR.\n\n{formatted.internal}")
+
+    return formatted
+
+
+def update_contributors(release_info: ReleaseInfo) -> None:
+    if release_info.branch == "main":
+        CONTRIBUTORS_PATH.write_text(
+            "Created as part of the release process.\n\n+ "
```

Comment:
nit, can you move the `"\n+ "` to its own line. It's really hard to miss the characters after, and just see the `"\n\n"`

Reply:
I'll rephrase the whole thing, although I'll note that this code was directly moved from the old script, and that optimising the formatting of a string used for code-generation that's checked into git (and thus surfaced in PR review) seems low value.